### DISCUSSION TITLE
NUMPAD 7

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#181] [Q&A]
It's strange that numpad 7 doesn't change the grid size, or is it? Does it do something else?

**Comment by Garux** [comment]
![image](https://github.com/Garux/netradiant-custom/assets/1734117/559d6dda-e4e2-4979-952d-ffe73731cf50)

**Comment by Hipshot-zz** [comment]
Ah yea, I see now, I was looking in the help section for this window for some reason.

But yea, the numpad does change the grid AND other things, but the grid thing isn't specified in the list there as an additional button. Old radiant had this, so I suppose it's more hardcoded into the program from olden times.

**Comment by Garux** [comment]
Numpad behaves interestingly in Qt; e.g. `num enter` doesn't work as regular `enter` alias, while numbers do; but it turns out they only do, when they are not bound to anything else. You can simply unbind them for grid size functionality.
